src/analyze/analyze-security.c

```diff
@@ -1824,6 +1830,33 @@ static int property_read_ip_address_allow(
         return sd_bus_message_exit_container(m);
 }
 
+static int property_read_ip_filters(
+                sd_bus *bus,
+                const char *member,
+                sd_bus_message *m,
+                sd_bus_error *error,
+                void *userdata) {
+
+        struct security_info *info = userdata;
+        char **l;
```

Comment:
needs to be freed. sd_bus_message_read_strv() allocates...

i.e. change this to:

```c
_cleanup_(strv_freep) char **l = NULL;
```

Reply:
Thanks